region/crateria/central.json

```diff
@@ -3793,6 +4056,33 @@
             {
               "id": 3,
               "strats": [
+                {
+                  "name": "Full Room Temporary Blue",
+                  "notable": false,
+                  "requires": [
+                    "canTemporaryBlue",
+                    "canCarefulJump",
+                    {"canComeInCharged": {
+                      "fromNode": 2,
+                      "framesRemaining": 180,
+                      "shinesparkFrames": 0
+                    }},
+                    {"or": [
+                      "canChainTemporaryBlue",
+                      {"and": [
+                        "canSpeedball",
```

Comment:
the speedball sounds hard. does it need a tricky jump?

Reply:
The speedball was the easiest of the three, just took some controlled hops with canSpringballBounce.  Thinking about it though, I only tried entering from the elevator room and had very low run speed.  I need to test run it while going faster.